JS Assignment/Function_assignment1.js

```diff
@@ -0,0 +1,29 @@
+// Assignment 1
+//Q1. Write a program to demonstrate how a function can be passed as a parameter to another function.
+
+function child(pqr){
+    console.log( pqr+10);
+  }
+  
+  function funmain(abc, child)
```

Comment:
Declare the names of Functions and Variables meaningful

Reply:
Okk Nagarjuna